<ui version="4.0" >
 <class>DataRange</class>
 <widget class="QWidget" name="DataRange" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>124</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <property name="horizontalSpacing" >
    <number>0</number>
   </property>
   <property name="verticalSpacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox2" >
     <property name="title" >
      <string>Data Range</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item row="2" column="5" >
       <widget class="QCheckBox" name="_doFilter" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis" >
         <string>Select this to average the skipped frames together.  Provides simple low pass filtering.</string>
        </property>
        <property name="text" >
         <string>Boxcar &amp;filter first</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QCheckBox" name="_doSkip" >
        <property name="toolTip" >
         <string>Save memory by sparsely sampling the data</string>
        </property>
        <property name="whatsThis" >
         <string>Select this to skip frames, rather than reading all of the data.  This is useful for reading very long but slowly changing data files.</string>
        </property>
        <property name="text" >
         <string>Read 1 sample per:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5" >
       <widget class="QCheckBox" name="_readToEnd" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis" >
         <string>Select this to read all of the data up to the end of the file.  </string>
        </property>
        <property name="text" >
         <string>&amp;Read to end</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Or</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="4" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Or</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="5" >
       <widget class="QCheckBox" name="_countFromEnd" >
        <property name="whatsThis" >
         <string>Select this to read data from the end of the file.  This is useful for scrolling displays when the file is being appended by another application.</string>
        </property>
        <property name="text" >
         <string>Co&amp;unt from end</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2" >
       <widget class="QSpinBox" name="_skip" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis" >
         <string>Number of frame to skip, if "Skip Frames" is selected.</string>
        </property>
        <property name="suffix" >
         <string> frames</string>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>2147483647</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Start:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis" >
         <string>Select this to average the skipped frames together.  Provides simple low pass filtering.</string>
        </property>
        <property name="text" >
         <string>Range:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2" >
       <widget class="QLineEdit" name="_start" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Input a number, or the format [[[[YYYY:]MM:]DD:]HH:]MM[.SS] for dates.</string>
        </property>
        <property name="whatsThis" >
         <string>The index of the first data read.  For ASCII files, this is the data row number in the file.  For frame based files, this is the first frame.</string>
        </property>
        <property name="text" >
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2" >
       <widget class="QLineEdit" name="_range" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Input a number, or the format [[[[YYYY:]MM:]DD:]HH:]MM[.SS] for dates.</string>
        </property>
        <property name="whatsThis" >
         <string>The amount of data to read into the vector.  For ASCII files, this is the number of data rows.  For frame files there may be more than one sample per frame.  Dates are input in the format [[[[YYYY:]MM:]DD:]HH:]MM[.SS].</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QComboBox" name="_startUnits" >
        <item>
         <property name="text" >
          <string>frames</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QComboBox" name="_rangeUnits" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <item>
         <property name="text" >
          <string>frames</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>_start</tabstop>
  <tabstop>_startUnits</tabstop>
  <tabstop>_countFromEnd</tabstop>
  <tabstop>_range</tabstop>
  <tabstop>_rangeUnits</tabstop>
  <tabstop>_readToEnd</tabstop>
  <tabstop>_doSkip</tabstop>
  <tabstop>_skip</tabstop>
  <tabstop>_doFilter</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
